src/main/java/filemanager/launcher/Launcher.java

```diff
@@ -3,13 +3,28 @@
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import filemanager.binder.FileServiceBinderModule;
-import filemanager.controller.Application;
+import filemanager.configuration.FileHandlerConfiguration;
+import filemanager.directorytracker.ScheduleFileDirectory;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Environment;
 
-public class Launcher {
-    public static void main(String[] args) {
-        Injector guice = Guice.createInjector(new FileServiceBinderModule());
-        Application application = guice.getInstance(Application.class);
-        application.runProgram();
+import java.util.concurrent.TimeUnit;
+
+public class Launcher extends Application<FileHandlerConfiguration> {
+    public static void main(String[] args) throws Exception {
+        new Launcher().run(args);
     }
 
+    @Override
+    public void run(FileHandlerConfiguration configuration, Environment environment) {
+        Injector guice = Guice.createInjector(new FileServiceBinderModule());
+        ScheduleFileDirectory trackerFileDirectory = guice.getInstance(ScheduleFileDirectory.class);
+        trackerFileDirectory.rootFolder = configuration.getRootFolder();
+        trackerFileDirectory.environment = configuration.getEnvironment();
+        trackerFileDirectory.outputPath = configuration.getOutputPath();
+        trackerFileDirectory.fileNamePattern = configuration.getFileNamePattern();
```

Comment:
Such usage of public fields is discouraged, please rewrite it to be handled by Guice. So that injector gave you fully-configured instance. There you may either use constructor for this purpose, or a bunch setters. Whatever you like more